universal-application-tool-0.0.1/app/services/question/QuestionService.java

```diff
@@ -51,4 +54,9 @@
   void archiveQuestion(Long id) throws InvalidUpdateException;
 
   void discardDraft(Long id) throws InvalidUpdateException;
+
+  ImmutableList<QuestionDefinition> getQuestionsForTag(QuestionTag tag);
```

Comment:
suggestion (blocking): please document this public method

Reply:
Done.